server/token.go

```diff
@@ -25,7 +25,9 @@ THE SOFTWARE.
 package server
 
 import (
-	"math/rand"
+	"crypto/rand"
```

Comment:
please, keep the changes to this file in the separated PR :)

Reply:
Ooops, sory